src/platform/Ameba/CHIPDevicePlatformConfig.h

```diff
@@ -0,0 +1,65 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *          Platform-specific configuration overrides for the chip Device Layer
+ *          on Ameba platform.
+ */
+
+#pragma once
+
+// ==================== Platform Adaptations ====================
+
```

Comment:
Could you add
```
#define CHIP_CONFIG_ERROR_CLASS 1
```
to ensure that nothing assumes `CHIP_ERROR` is an int? (I to a quick look through and didn't notice any problems.)

Reply:
Thanks for the review, the suggested change has been applied!